src/Mod/Path/Gui/Resources/panels/PageOpVcarveEdit.ui

```diff
@@ -95,14 +95,14 @@
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
-         <string>Threshold</string>
+         <string>Filter Colinear lines</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="threshold">
```

Comment:
Do you want to leave the spinbox's name as "threshold" here?

Reply:
Not gonna let me get away with being lazy, are ya?    Fine.  Fixed.